packages/@sanity/schema/src/legacy/types/blocks/defaults.ts

```diff
@@ -1,6 +1,8 @@
 export const DEFAULT_LINK_ANNOTATION = {
   type: 'object',
   name: 'link',
+  title: 'Link',
+  i18nTitle: 'inputs.portable-text.annotation.link',
```

Comment:
nit: definitely not worth blocking over, but should `i18nTitle` be `i18nTitleKey`? It's not actually the title, but the key to look up the title.

Reply:
Good point. Will rename.